src/index.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-unused-vars */
+import _ from 'lodash';
+import './style.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const taskArr = [];
+const taskContainerUl = document.getElementById('list');
+
+class Task {
+  constructor(description = ' ', completed = false, index = (taskArr.length + 1)) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const a = new Task('task a');
+taskArr.push(a);
+const b = new Task('task b');
+taskArr.push(b);
+const c = new Task('task c');
+taskArr.push(c);
+const d = new Task('task d');
+taskArr.push(d);
```

Comment:
## * I suggest you refer back to the project requirement. an array of some simple to do tasks (array of objects) should be created kindly make the necessary adjustments:
![image](https://user-images.githubusercontent.com/52497246/125534185-c5d39d4c-68c4-4393-aa74-c3f1f4b62626.png)

Reply:
It is an array of objects please review the code

Reply:
* Am so sorry i missed the array 🙏 it was a mistake